### PR TITLE
Add capabilities to task drivers' pages

### PR DESCRIPTION
**Pull request by kobayashi**
This add table of capabilities to each task driver page. #8402

### CODE REVIEW

File: website/pages/docs/drivers/java.mdx [hunk 1]
```diff
@@ -97,6 +97,15 @@ task "web" {
...
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| Exec | false |
+| FSIsolation | FSIsolationNone  |
+| NetIsolationModes | NetIsolationModeHost, NetIsolationmodeGroup |
+| MountConfigs | MountConfigSupportNone |
```
Comment (angrycub) [inline website/pages/docs/drivers/java.mdx:107]: @tgross are these true on Linux?  I think that the macOS version of the Java driver tosses FS Isolation, but I thought the Linux version is like exec+java

File: website/pages/docs/drivers/java.mdx [hunk 2]
```diff
@@ -97,6 +97,15 @@ task "web" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| Exec | false |
+| FSIsolation | FSIsolationNone  |
```
Comment (tgross) [inline website/pages/docs/drivers/java.mdx:105]: On Linux the Java capabilities are slightly different: https://github.com/hashicorp/nomad/blob/v0.12.0/drivers/java/driver.go#L98-L101

File: website/pages/docs/drivers/exec.mdx [hunk 3]
```diff
@@ -77,6 +77,16 @@ task "example" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| SendSignals | true  |
+| Exec | true |
+| FSIsolation | FSIsolationImage |
```
Comment (tgross) [inline website/pages/docs/drivers/exec.mdx:86]: Should be chroot: https://github.com/hashicorp/nomad/blob/v0.12.0/drivers/exec/driver.go#L82

File: website/pages/docs/drivers/java.mdx [hunk 4]
```diff
@@ -97,6 +97,15 @@ task "web" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| Exec | false |
```
Comment (tgross) [inline website/pages/docs/drivers/java.mdx:104]: We're missing `SendSignals` as `false`.

File: website/pages/docs/drivers/exec.mdx [hunk 5]
```diff
@@ -77,6 +77,16 @@ task "example" {
...
+| NetIsolationModes | NetIsolationModeHost, NetIsolationmodeGroup |
```
Comment (tgross) [inline website/pages/docs/drivers/exec.mdx:87]: We should include `MustInitiateNetwork | false` everywhere it's not explicitly set to `true` by default.

File: website/pages/docs/drivers/qemu.mdx [hunk 6]
```diff
@@ -101,6 +101,15 @@ task "virtual" {
...
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| SendSignals | false |
+| Exec | false |
+| FSIsolation | FSIsolationImage |
+| MountConfigs | MountConfigSupportNone |
```
Comment (tgross) [inline website/pages/docs/drivers/qemu.mdx:111]: We should include `| NetIsolationModes | none |` here

**Comment by kobayashi** [comment]
Thank you for the review @tgross !
I just modified tables.

**Pull request merged**